crates/routes/src/images.rs

```diff
@@ -221,7 +221,10 @@ async fn purge(
   }
 
   // Add the API token, X-Api-Token header
-  client_req = client_req.header("x-api-token", pictrs_config.api_key);
+  let pictrs_api_key = pictrs_config
+    .api_key
+    .ok_or_else(|| LemmyError::from_message("pictrs_api_key_not_provided"))?;
+  client_req = client_req.header("x-api-token", pictrs_api_key);
```

Comment:
I dont really understand what this method is for. And if it is necessary, why not simply have it call purge_image_from_pictrs()?

Reply:
This one is unused currently. So these routes are ones that forward requests to pictrs ( IE uploads, gets, etc) , whereas the one in `api_common` is a helper method that takes in a image_url directly, rather than through URL params.

Reply:
Then we can remove it, right?

Reply:
We should keep it IMO, because otherwise if we want to do *manual* image purge commands via curl won't work.